### PR TITLE
Multiple argument for service start/stop commands

### PR DESCRIPTION
**Pull request by antho1404**
It is now possible to call the commands `service start` and `service stop` with multiple arguments and not only one.

```bash
mesg-core service stop sid1 sid2 sid3...
```

### CODE REVIEW

File: commands/service_list.go [hunk 1]
```diff
@@ -30,21 +30,23 @@ To have more details, see the [detail command](mesg-core_service_detail.md).`,
 		Args:    cobra.NoArgs,
 		RunE:    c.runE,
 	})
+	c.cmd.Flags().BoolVarP(&c.quiet, "quiet", "q", c.quiet, "Display only the service hashes")
```
Comment (krhubert) [inline commands/service_list.go]: I would change `quiet` to `--format` as this one is what we want.

Quiet is grate flag to be implemented globally and suppress all progress/messages that are not relevant for final output

Comment (antho1404) [reply commands/service_list.go]: i will remove it from this PR, it needs more thoughts and it should have been two different PR anyway

**Comment by antho1404** [comment]
I removed the `-q` on the list command, we need to discuss more about that before any implementation. This PR is only about multi arguments for start and stop commands

File: commands/service_start.go [hunk 2]
```diff
@@ -19,25 +19,29 @@ func newServiceStartCmd(e ServiceExecutor) *serviceStartCmd {
...
-		return err
+	for _, arg := range args {
+		var err error
+		// build function to avoid using arg inside progress
+		fn := func(serviceID string) func() {
+			return func() {
+				err = c.e.ServiceStart(serviceID)
+			}
+		}(arg)
+		pretty.Progress("Starting service...", fn)
```
Comment (NicolasMahe) [inline commands/service_start.go]:
```suggestion
		pretty.Progress(fmt.Sprintf("Starting service %q...", arg), fn)
```

File: commands/service_stop.go [hunk 3]
```diff
@@ -18,21 +18,26 @@ func newServiceStopCmd(e ServiceExecutor) *serviceStopCmd {
...
-		return err
+	for _, arg := range args {
+		var err error
+		// build function to avoid using arg inside progress
+		fn := func(serviceID string) func() {
+			return func() {
+				err = c.e.ServiceStop(serviceID)
+			}
+		}(arg)
+		pretty.Progress("Stopping service...", fn)
```
Comment (NicolasMahe) [inline commands/service_stop.go]:
```suggestion
		pretty.Progress(fmt.Sprintf("Stopping service %q...", arg), fn)
```

**Pull request merged**